exporters/slow_log.go

```diff
@@ -432,3 +434,11 @@ func listsToMap(k, v []string) map[string]string {
 	}
 	return m
 }
+
+func errListsToMap(k, v []uint64) map[uint64]uint64 {
+	m := map[uint64]uint64{}
+	for i, e := range k {
+		m[e] = v[i]
+	}
+	return m
+}
```

Comment:
I think we should move that tool to pmm-agent repository soon – it already has a different version of slow log file parser, for example.

Reply:
Sure, I'll do.